src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/PrisonerSearchService.kt

```diff
@@ -21,6 +21,7 @@ class PrisonerSearchService(
 ) {
   companion object {
     val log: Logger = LoggerFactory.getLogger(this::class.java)
+    const val RESULT_HITS_MAX = 1000
```

Comment:
Wonder if this at some point should be config to allow for tests to tweak it etc

Reply:
yeah - wondered about that - but it took a long time to get the int tests working. Hopefully I'll get chance to do that